1907-01-21.xml

```diff
@@ -1494,13 +1497,979 @@
+                        heir possessors are only just beginning to dirt Ve should not be surprised
+                        to see the Death-utiea double their yield during the next wenty years.</p>
+                </div>
+            </div>
+            </div>
+            <div type="page" n="3">
+            </div>
+            <div type="masthead"> 
+                <head>THE EGYPTIAN GAZETTE, MONDAY JANUARY 21, 1907</head>
+            </div>
+            <cb n="1"/>
+            <div type="section" element="local">
+                <head>LOCAL AND GENERAL</head>
+                <div type="item"><p>Tour Eiffel Music Hall.</p>
+                <p> The "Sporting Trin" of English singers and dancers made a successful debut at
+                    this theatre last night.</p></div>
+                <div type="item"><p>Quarantine In Turkey.</p>
+                <p>A telegram from Constantinople states that arrivals from Port Said are subject to
+                    a medical examination.</p></div>
+                <div type="item"><p>Council of Ministers. </p>
+                <p>The Council of Ministers will meet on Tuesday aftenoon at Aberdeen Palace under
+                    the presidency of the Khedive.</p></div>
+                <div type="item"><p>The Weather.</p>
+                <p>Owing to the high sea running outside the harbour pass, bessels were unable to
+                    enter the port this afternoon. The black flag is flying over Komel Nadoura,
+                    which indicates that the pilot cannot leave the harbour.</p></div>
+                <div type="item"><p>Visit of Japanese Squadron.</p>
+                <p>A Japanese Squadron which is on its way to Great Britain on an official visit is
+                    due at Suez in the early party of February. It is not yet known whether the
+                    Admiral and his staff will go to Cairo to be received by the Khedive.</p></div>
+                <div type="item"><p>Five Days on a Desert Island.</p>
+                <p>An Ottoman sailing ship which lately left this port for Scio, in ballast was
+                    surprised by a squall and sunk off the isle of Poffs. The master and crew of six
+                    reached shore in the ship's boat. The islet is barren and for five days they
+                    were without food. They were picked up by a saiiling ship from Simi.</p></div>
+                <div type="item"><p>Kourban Beyram.</p>
+                <p>The Ministry of War has issued orders for salnes to be fired on Wednesday
+                    afternooon, and on each of the following four days, on the occasion of Kourban
+                    Beyram The Khedive will attend Beyraum prayers early on Thursday morning at Koub
+                    eh Mosque. A salute of 21 guns will the be fired from the Citadel and Abbassieh
+                    barracks.</p></div>
+            </div>
```

Comment:
this `</div>` ends the local and general section, and should come after the next item